Client/mods/deathmatch/logic/luadefs/CLuaPlayerDefs.cpp

```diff
@@ -309,6 +311,11 @@ int CLuaPlayerDefs::GetPlayerWantedLevel(lua_State* luaVM)
     return 1;
 }
 
+const std::uint8_t CLuaPlayerDefs::GetPlayerScriptDebugLevel() noexcept
```

Comment:
const here is unnecessary and doesn't make much sense

Reply:
Thanks